src/py-opentimelineio/opentimelineio/plugins/manifest.py

```diff
@@ -202,10 +202,10 @@ def load_manifest():
 
     The order of loading (and precedence) is:
 
-       1. manifests specfied via the ``OTIO_PLUGIN_MANIFEST_PATH`` variable
-       2. builtin plugin manifest
-       3. contrib plugin manifest
-       4. ``setuptools.pkg_resources`` based plugin manifests
+       1. manifests specified via the ``OTIO_PLUGIN_MANIFEST_PATH`` variable
+       2. ``setuptools.pkg_resources`` based plugin manifests
```

Comment:
I know it was there before, but technically it's not "setuptools" based. It should say "Entrypoint based plugins".

Reply:
maybe this?
2. ``pkg_resources`` Entrypoint based plugin manifests

Reply:
This is still inaccurate. `setuptools.pkg_resources` is just an API that can find and load entry points points. There is nothing setuptools specific when it comes to entry points. It just happens that they were introduced by setuptools long ago and is now a packaging specification (see https://packaging.python.org/en/latest/specifications/entry-points/).

Python 3 even has [a builtin API](https://docs.python.org/3/library/importlib.metadata.html) to support them, like mentioned in https://setuptools.pypa.io/en/latest/pkg_resources.html. `importlib.metadata` also has a backport package in PyPI, which we just happen to not be using.

Reply:
Okay how about I just change it to 
```
2. Entrypoint based plugin manifests
```

Reply:
Seems good to me.